src/Xamarin.Android.Build.Tasks/Utilities/NativeAssemblyGenerator/NativeAssemblerContextDataProvider.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+namespace Xamarin.Android.Tasks
+{
+	class NativeAssemblerStructContextDataProvider
+	{
+		public virtual string GetComment (object data, string fieldName)
+		{
+			return String.Empty;
+		}
+
+		public virtual uint GetMaxInlineWidth (object data, string fieldName)
```

Comment:
What is an "inline" width?

Reply:
Get maximum width of data buffer allocated inline (that is not pointed to)